Mage.Sets/src/mage/cards/r/RaziaBorosArchangel.java

```diff
@@ -81,7 +81,7 @@ class RaziaBorosArchangelEffect extends RedirectionEffect {
     protected MageObjectReference redirectToObject;
 
     public RaziaBorosArchangelEffect(Duration duration, int amount) {
-        super(duration, 3, UsageType.ONE_USAGE_ABSOLUTE);
+        super(duration, 3, UsageType.ACCORDING_DURATION);
```

Comment:
Can we fix the mixed use of constant `3` and variable `amount` while we're at it?

Reply:
I think that can fit into this PR. Done.